flight/protoMap.py

```diff
@@ -0,0 +1,267 @@
+    altitude: int
+    focal_length: float
+
+    Returns
+    -------
+    path: List[Tuple[float, float]]
+        The path that the drone will take through the map
+    """
+
+    # Takes in tuple(start location, map height), altitude, and focal_length]
+    # Assumes altitude and focal_length doesn't change
+    # Returns flight path of drone to snap images of map
+
+    # Split info into center of map location and map height
+    center_map: Tuple[float, float] = info[0]
+    map_h: float = info[1]
+
+    # Calculate the width and height of camera image
+    cam_w, cam_h = image_area(altitude, focal_length)
+    print("Camera Width:", cam_w)
+    print("Camera Height:", cam_h)
+
+    # Determine map width using MAP_H and aspect ratio 16:9
+    # Units: ft
+    map_w: float = (16 * map_h) / 9
+
+    print("Map Width:", map_w)
+    print("Map Height:", map_h)
+
+    # Find center of each piece consider OVERLAP
+    ft_waypoints: List[Tuple[float, float]] = find_centers(
+        map_w, map_h, cam_w, cam_h, OVERLAP
+    )
+    print("Num of waypoints:", len(ft_waypoints))
+
+    # Convert centers to lat, long coordinates
+    coord: List[Tuple[float, float]] = coordinates(ft_waypoints, info[0], map_w, map_h)
+    print("Coord:", len(coord))
+
+    # Flight path algorithm
+    pass
+
+
+def image_area(altitude: int, focal_length: float) -> Tuple[float, float]:
+    """
+    Takes in the altitude of the drone and the zoom (focal_length) in order to determine
+    the area of the image captures by the drone
+
+    Parameters
+    ----------
+    altitude: int
+    focal_length: float
+
+    Returns
+    -------
+    (cam_w, cam_h): Tuple[float, float]
+
+    Notes
+    -----
+    fov is the Field of View of the image area
+    fovH is the horizontal field of view distance
+    fovV is the vertical field of view distance
+    """
+
+    fov: float = 2 * math.atan((SENSOR_W / (2 * focal_length)))
+    print("FOV:", fov)
+
+    fovH: float = SENSOR_W / SENSOR_D * fov
+    fovV: float = SENSOR_H / SENSOR_D * fov
+
+    print("fovH:", fovH)
+    print("fovV:", fovV)
+
+    cam_w: float = 2 * altitude * math.tan(fovH / 2)
+    cam_h: float = 2 * altitude * math.tan(fovV / 2)
+
+    return (cam_w, cam_h)
+
+
+def find_centers(
+    map_w: float,
+    map_h: float,
+    cam_w: float,
+    cam_h: float,
+    overlap: float,
+) -> List[Tuple[float, float]]:
+    """
+    This function will split the map into even sections accounting for the image area
+    and overlap for image stitching. It will start at the top left corner and move
+    a determined distance from from the top left corner to the center of a section
+    for every section.
+
+    Parameters
+    ----------
+    map_w: float
+    map_h: float
+    cam_w: float
+    cam_h: float,
+    overlap: float
+
+    Returns
+    -------
+    waypoints: List[Tuple[float, float]]
+    """
+
+    # Return a 2d list of center locations for each image
+    # List of way points
```

Comment:
Have this information in the docstring for the function; specify what these waypoints are for in a short description for the return type, underneath the structure and type hint:
`waypoints: List[Tuple[float, float]]
      list of waypoints for mapping flight algorithm`